kml.cc

```diff
@@ -548,12 +548,12 @@ kml_wr_deinit(void)
   delete oqfile;
   oqfile = NULL;
 
-  if (posnfilenametmp) {
+  if (!posnfilenametmp.isEmpty()) {
 #if __WIN32__
-    MoveFileExA(posnfilenametmp, posnfilename,
+    MoveFileExA(qPrintable(posnfilenametmp), qPrintable(posnfilename),
                 MOVEFILE_REPLACE_EXISTING);
 #endif
-    rename(posnfilenametmp, posnfilename);
+    QFile::rename(posnfilenametmp, posnfilename);
```

Comment:
I'd bet we don't actually need this abstraction as QFile::rename() should work everywhere, but this is fine.